Installation issue: graphviz

<!-- Thanks for taking the time to report this build failure. To proceed with the report please:

1. Title the issue "Installation issue: <name-of-the-package>".
2. Provide the information required below.

We encourage you to try, as much as possible, to reduce your problem to the minimal example that still reproduces the issue. That would help us a lot in fixing it quickly and effectively! -->

### Steps to reproduce the issue

<!-- Fill in the exact spec you are trying to build and the relevant part of the error message -->
This error occurs for versions 2.46.0, 2.42.4, 2.38.0 but not 2.44.1
```console
$ spack install graphviz@2.46.0
==> Warning: Missing a source id for graphviz@2.46.0
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libsigsegv-2.12-sgtkkhybmq7gd7zhyvvzprnr5lnof3g2
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/berkeley-db-18.1.40-pgrytburszenp57k2q4osv5nkxwngalf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pkgconf-1.7.3-3id2k25wjdrnrz6bzxsruf7vg2ukytx7
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libiconv-1.16-gegj5jbht56ykaif5omnsriv2f46eqnl
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xz-5.2.5-bla6c6jasr7k52yds2h2ymondiazarcc
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/zlib-1.2.11-u3tjtvurvda5bde33uduo432cg4xgbvj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libbsd-0.10.0-jwvv7vensddoin2tuafatzalnwsphzxo
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libffi-3.3-uinxpvmyxs3kwxjczu5rqdbrowxyzljd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libuuid-1.0.3-enrad552w66dkgftzrdoinbrcfrmlpkd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pcre-8.44-cb3xri3cfopbei3nvowc5pdsvalhy66n
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/util-macros-1.19.1-jvu2uf3u72sly5xrjdffgtxozhpf4232
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gperf-3.1-c6flvoxsr7xmuy6yhqfmlnm3qpzfbwqm
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libpthread-stubs-0.4-twhifhfdvskne4bqhlrjem274wupe7jp
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xcb-proto-1.13-3zopudjf77txg37seb7tcuf5kay6cd42
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/docbook-xml-4.5-bs4kifbercjv2pz6pgvttwvto6qhw5ad
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/nasm-2.15.05-tzyyfl6qgidoyk46psepkdd4xvlft5n5
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/autoconf-archive-2019.01.06-i4kjto3oup5gurrfus6utjqk5geeoce4
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/sed-4.2.2-zyvhr3gpugk3hbfketjgxw2qzt5x42am
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/m4-1.4.18-7ygnb5agc4rzhqnpamsxw7qsgfrx6vsc
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/ncurses-6.2-d4uhyy4ixog5q2qjcnn6nhmut4btyt25
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/diffutils-3.7-lqwas65uyzocbgzo6z2oq526agp6mipw
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/tar-1.32-24gfjty3h44udzm45xswsnwqbun255x7
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libpng-1.6.37-fqcar4j4qoqhwpln4txnnqvrrxr6uswf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxml2-2.9.10-4xuwlgefwqhde3kfks2xp4dixf6jg63f
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/expat-2.2.9-jnq5tcra4f2nxcegt6l4hhcqffnlah6l
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/swig-4.0.2-nzziqewo36dzshk7mfwngb3555lxp2ec
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/inputproto-2.3.2-ukdmoqruayow6ezjchf5e3xwepdw2ppj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/recordproto-1.14.2-qibsgh62chhzs3l6zihaexdgl2t6pwcf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xtrans-1.3.5-v7w72elpcuxbrjdqdcestvfemqef3v7o
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/fontsproto-2.1.3-wh4h2omcikybpn5caqbos2azegeblfxh
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/kbproto-1.0.7-3d3oxvsgjg2yp6jmj2kusngebklqmbbb
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/renderproto-0.11.1-7g6a5lbnp6uauztxqin7xn7aaf5desxh
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/fixesproto-5.0-7zfwmw6r7dslb7qcmysoirty6vtv5jud
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xproto-7.0.31-lagnefofws2za2tjyyisopyn3ubjq6hf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xextproto-7.3.0-qn5nbiarhg6mnycmows6lllznyidmugo
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glproto-1.4.17-pf4ajxwvd74ww7r6ac562ip3eu5bkhyj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/docbook-xsl-1.79.2-k5o4jahdyazraqhonbv7rqviy6bwqoq6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libtool-2.4.6-cxbrquzht2yopb7wri2wzkynh5xdl2u7
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/readline-8.0-bz6z2lhrx6lldfd5enf7fvkfvsp7el77
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libedit-3.1-20191231-jrtgbd2oc4hry6wnzxgt2egtg7qpykor
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/bzip2-1.0.8-x3bcddi6guspno4lueedcr7nl3eunzwd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pixman-0.40.0-cjjlunp5xgxyz3ljnl7noclc6zdytess
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libfontenc-1.1.3-b2vwwvpzcupcjsicmuoj3qvh3dm62va2
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxdmcp-1.1.2-eqbxluood4ohvokmnvzvmjf2kv7g6gah
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxau-1.0.8-ct7fnne5xumvy4zmqam6jnqlhgr2w7fq
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libice-1.0.9-6od4hpykt35l7kw7svstktxus7iokjxm
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libpciaccess-0.16-2zmcrpripc4n5ud6j3t66w5qfi6fn5it
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/sqlite-3.33.0-ofbj2qjq6fshwldmfphfun6y7b7wsm73
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gdbm-1.18.1-nldw7xaht5lastlonsacknvh4iajsi2e
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/freetype-2.10.1-muu5rmlexvgom5t5y73nftqrq5zlehhl
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxcb-1.13-uczgtz6ber7a7z64zwpxo2ufs5pfqlva
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libsm-1.2.3-qn2i6uxi4gfawuzulpua4edezyvt7ek6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/hwloc-2.2.0-rikeju6buiavjifjo7qobn6sbhvtpmwu
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-5.30.3-whlg25cpagc4yel3mkooocqgm7kroan4
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxfont-1.5.2-l4jufqz3fwkft74swmyz4qqqqprii7yn
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/mkfontscale-1.1.2-nax7dcgtogshwp62fw2lvlbwkxgldydl
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/binutils-2.35-l3fgoglwls4uo636onhlzlf6saq6t6rz
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/texinfo-6.5-lzpi3yafmllhoyv2xu4amrhxthdvn73s
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/autoconf-2.69-dx45n64n5dhjmxjyq7rc52ijw6mnrkak
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-extutils-helpers-0.026-5rzw6i3kd7j4vzd322v42z6mrmwgrbu6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-html-tagset-3.20-pswfnyuddqb4pkcipjirj6j5gbdbebf2
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/openssl-1.1.1g-7khf64bbk2xrrtc76e4va75vsww3jwnc
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-io-html-1.001-iu7ekjb7gthqunkwnsa3sjaem37x2ato
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-test-needs-0.002005-qbrx4qyri3z55pfsi6s7e5oa42gqhseg
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libx11-1.6.7-grhex6prg7mxfbgwlmg7efamkb6mgrup
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-lwp-mediatypes-6.02-6iuvppjys5vx3jwy7z7gnea3vbxkud53
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-extutils-installpaths-0.012-h4y2umlll37tjn6xk2qiblobehylziqq
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-try-tiny-0.28-ttkbsdzlodv57hj5iprsqsggcgd4vtbl
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-data-dumper-2.173-73uh44adapth2l7jxab3fzboz6o6cppw
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-encode-locale-1.05-ci3fe4p57a6n5aihmayzruswtw7roj2p
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/help2man-1.47.11-kw7ipynslnluqvbvwkrq5uwnfnd5posm
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-extutils-config-0.008-bppuzyjx5hr3jfts36qavcteaybu7fnt
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-http-date-6.02-wlvdoa3hz2un7wnshj7v4n2qg54jey7q
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-module-build-0.4224-rskf5o755ng3m7djs3jf7i6v3b6ozkpg
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/bdftopcf-1.0.5-pvbd3a7t3y7prcd5np3gnsnrgnmvdoqk
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/mkfontdir-1.0.7-enw6g4sz5y5l646jxffki4exwnyqhqli
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gmake-4.2.1-oj44mzjlscklgh7nxadpaahk27kqfmva
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/automake-1.16.2-jevxt3ujiqvh6fpsjzn655qrblqf4t7o
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-html-parser-3.72-ko252rhoz3xwhg4kvila3iipyvw4wodi
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/cmake-3.18.2-pgyr4o3q3xj3ppth3eyhl7kb2ltcvg4g
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/python-3.8.6-tj37yonzdw2zzlt5abf3flwoyiypfxcd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-uri-1.72-7tus35c2zp4ety5beyv32yqt6u4xl3ge
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxfixes-5.0.2-rctw3igvjluoslxoo63jl3lzc44swauf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxrender-0.9.10-gpvjhqfz2gpq5ksyx2u3t2og7wsaswm3
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxext-1.3.3-cp45ih7szzrj7ft6isljb742bnptcclj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/bison-3.6.4-a4vfdlubidnzayscjobbkguvsbbsph45
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-file-listing-6.04-knej4tq2l7ijq3lc2od4wnpobadscynx
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-module-build-tiny-0.039-m4dpyfiwhio6lufk4usgrhhntqn3xgzy
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/findutils-4.6.0-u4upkyiy7ni2aq6dxmppxnpnpvrbm7tq
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gmp-6.1.2-gmw7p2ssalvzpgc2k6bi26z4tp5apbwa
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/font-util-1.3.2-p52vtzxqgbj6wtqrykybjb7mxutmlmsv
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libjpeg-turbo-2.0.4-v7xumfx325ejh5iifvmlapoj4h5xalnc
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/ninja-1.10.1-qzlg5xsltc5w5znkdcib2ltvca5brcax
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/icu4c-67.1-7xgxsssk6qacmawokelucthykixs3v7r
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/py-setuptools-50.1.0-ktu7ulphomwn5mfti2rpuycou7q6dpra
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-http-message-6.13-z56jwsjo3fvhy5nojvidli2moaa7bjbd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-www-robotrules-6.02-67vzchnoroa4mb52wzay6o7bgge7ulqg
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-net-http-6.17-3jqto4vv2mum4jv7xccvsnpk5cnsv4ax
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxi-1.7.6-ek7gx6x4b5b3lqurshdgrissfdv4vgbr
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/flex-2.6.4-nrogoojtncavetefx4zgijl63itfkwve
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/mpfr-4.0.2-el56i5qd2b6qbzvs44dezolwgpcxn2kd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/fontconfig-2.13.1-an5w2kywpq7wzxaazpejayz5xnwiqumn
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libtiff-4.1.0-vsmpvspgkhm6y3kilybcuu3gpt3ctmhl
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/z3-4.8.7-dsdq5ytm76feylwshzjm2uwwizuncmzb
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/py-markupsafe-1.1.1-jngg52bzbuqn62btwwc74kmijfjkcnbq
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/meson-0.55.1-m5jqmdvsdvcjlhgw3jdm4vgydklaj7hc
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-http-daemon-6.01-ljggdx4i6rrbtndbqkfkbpr7xznghnmt
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-http-negotiate-6.01-4ey6qlez44hwxtwqbbcmc73jcx6z46fj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-http-cookies-6.04-6pvbaroxyqjmdks4ub3ed5cev7aleju7
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxtst-1.2.2-3dnkgm4fyjho2nq4mc53mqx3idnzankg
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gawk-5.0.1-2pfdoyvhu7qweo5vbdecnneefppa7hf3
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxft-2.3.2-qb3jg5hwea5au2e5d2znq4kgrjgylcvh
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/lcms-2.9-cr6qgoh6ttjt223jgzfktjc7papk77pv
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/llvm-10.0.1-de7qfqd5pmo3nki7lmaz4qck74rm76wy
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/py-mako-1.0.4-4o3z52crf6amdsjospdee3mhbz7qieqy
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.64.5-vnvgr6lmfyv3njzgdog2wtkx3wopolnm
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-libwww-perl-6.33-unzxehbydykqz5645a7qvumj253l7xoz
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libgpg-error-1.37-36qlxktdkcyaxceqno3cf3xkspuasyr6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/mesa-18.3.6-wgj4y5nhrdoqt5bbrn2wuqbcxkajpdea
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/dbus-1.12.8-bqdo6o7hiffe7tqga6smfgqz7p5q7ss6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/cairo-1.16.0-xlt7vsc2yw4u5ckhqmskoleatrptiocj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-xml-parser-2.44-yi6s2hqdqoho7uhxwvmiqjqps3bjir34
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libgcrypt-1.8.5-4mjm2hmuz57p2mxihflkrsngesbe6oam
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libepoxy-1.4.3-nhjn36mjgh6rk2ii2g6e7yjfk5ds3fwy
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/harfbuzz-2.6.8-th44s3rc6wzm7bjnsqz6yxwwuvnfliw6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gobject-introspection-1.56.1-ei5nrb5khrum45xyabdfsrds3zuscgav
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/intltool-0.51.0-nl3rtvog3u5vw4zkolbuogtpl5czjp56
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxslt-1.1.33-fcpv73nsmrqb6uukluisojlyiolxhtzh
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/atk-2.36.0-kheraeqnhwbh4en5wuedplleavdspzok
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/at-spi2-core-2.36.0-oljiai44nfgcrw2euc6k42mrc6isy2qf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pango-1.41.0-75ta3amlfb7we7nm346xnq6rk3hpm5um
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/shared-mime-info-1.9-bavmjwsw2nht7cextrt6fj7pvh4a2gw2
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/at-spi2-atk-2.34.2-pv44tiwb3n6tlviscd22w42ho3sn6ryv
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gdk-pixbuf-2.40.0-xvg2sgvq6sk2cwvhtyvvr2yirxd34n7w
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gtkplus-3.20.10-sylbwm5cpsz56cbyguugxfdtuti37tlg
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/ghostscript-9.50-yx5ayvqs56sjy546oevwjrq5j7exf5s2
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/groff-1.22.3-nthkaedr45rc5vvxlvfw6cpdgzabjofp
==> Error: Can't extrapolate a URL for version 2.46.0 because package graphviz defines no URLs

```

### Information on your system
* **Spack:** 0.15.4-1446-08e30a2
* **Python:** 3.6.8
* **Platform:** linux-centos7-sandybridge


<!-- Please include the output of `spack debug report` -->

<!-- If you have any relevant configuration detail (custom `packages.yaml` or `modules.yaml`, etc.) you can add that here as well. -->

### Additional information
There is no build-out.txt or build-env.txt as it failed before starting to install.
<!-- Please upload the following files. They should be present in the stage directory of the failing build. Also upload any config.log or similar file if one exists. -->


<!-- Some packages have maintainers who have volunteered to debug build failures. Run `spack maintainers <name-of-the-package>` and @mention them here if they exist. -->

### General information

<!-- These boxes can be checked by replacing [ ] with [x] or by clicking them after submitting the issue. -->
- [x] I have run `spack debug report` and reported the version of Spack/Python/Platform
- [x] I have run `spack maintainers <name-of-the-package>` and @mentioned any maintainers
- [ ] I have uploaded the build log and environment files
- [x] I have searched the issues of this repo and believe this is not a duplicate

Now I get a new error with the install. This happens with both 2.42.4 and 2.38.0. Let me know if you want the build txt. 

```
spack install graphviz@2.42.4+doc
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libiconv-1.16-gegj5jbht56ykaif5omnsriv2f46eqnl
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pkgconf-1.7.3-3id2k25wjdrnrz6bzxsruf7vg2ukytx7
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xz-5.2.5-bla6c6jasr7k52yds2h2ymondiazarcc
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/zlib-1.2.11-u3tjtvurvda5bde33uduo432cg4xgbvj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/berkeley-db-18.1.40-bhm5krr5wthfvrkrxdrsptxlf3xjn3x3
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libsigsegv-2.12-sgtkkhybmq7gd7zhyvvzprnr5lnof3g2
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libbsd-0.10.0-jwvv7vensddoin2tuafatzalnwsphzxo
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libffi-3.3-uinxpvmyxs3kwxjczu5rqdbrowxyzljd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pcre-8.44-cb3xri3cfopbei3nvowc5pdsvalhy66n
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/util-macros-1.19.1-jvu2uf3u72sly5xrjdffgtxozhpf4232
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gperf-3.1-c6flvoxsr7xmuy6yhqfmlnm3qpzfbwqm
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libpthread-stubs-0.4-twhifhfdvskne4bqhlrjem274wupe7jp
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xcb-proto-1.14.1-gnedyvmp7ebxordhaaciysi5hyomt7q6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/docbook-xml-4.3-acfea4zzun2dmtz3r6ixmp73qjrpsdzu
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/nasm-2.15.05-tzyyfl6qgidoyk46psepkdd4xvlft5n5
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/autoconf-archive-2019.01.06-i4kjto3oup5gurrfus6utjqk5geeoce4
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/sed-4.2.2-zyvhr3gpugk3hbfketjgxw2qzt5x42am
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/diffutils-3.7-lqwas65uyzocbgzo6z2oq526agp6mipw
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/tar-1.32-24gfjty3h44udzm45xswsnwqbun255x7
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/ncurses-6.2-d4uhyy4ixog5q2qjcnn6nhmut4btyt25
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/util-linux-uuid-2.36-azx7oni5ml5zz66c57wa7oytydgbyzz5
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxml2-2.9.10-4xuwlgefwqhde3kfks2xp4dixf6jg63f
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libpng-1.6.37-fqcar4j4qoqhwpln4txnnqvrrxr6uswf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/m4-1.4.18-7ygnb5agc4rzhqnpamsxw7qsgfrx6vsc
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/expat-2.2.10-ajprkrtqt2cnzeq2h3su2kgcgbfm73zv
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/swig-4.0.2-nzziqewo36dzshk7mfwngb3555lxp2ec
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glproto-1.4.17-pf4ajxwvd74ww7r6ac562ip3eu5bkhyj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xtrans-1.3.5-v7w72elpcuxbrjdqdcestvfemqef3v7o
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/renderproto-0.11.1-7g6a5lbnp6uauztxqin7xn7aaf5desxh
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xextproto-7.3.0-qn5nbiarhg6mnycmows6lllznyidmugo
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/randrproto-1.5.0-5cu7unt6hrc6mqtzku6ua4mugy3bpdpa
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xproto-7.0.31-lagnefofws2za2tjyyisopyn3ubjq6hf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/inputproto-2.3.2-ukdmoqruayow6ezjchf5e3xwepdw2ppj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/fixesproto-5.0-7zfwmw6r7dslb7qcmysoirty6vtv5jud
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/kbproto-1.0.7-3d3oxvsgjg2yp6jmj2kusngebklqmbbb
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/recordproto-1.14.2-qibsgh62chhzs3l6zihaexdgl2t6pwcf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/fontsproto-2.1.3-wh4h2omcikybpn5caqbos2azegeblfxh
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/docbook-xsl-1.78.1-shvyabphu6uakhlf2h5elsrtzca7e45m
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/bzip2-1.0.8-x3bcddi6guspno4lueedcr7nl3eunzwd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/readline-8.0-bz6z2lhrx6lldfd5enf7fvkfvsp7el77
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libedit-3.1-20210216-vi47xrruba5m6sbt37voix3zhvqgdr4p
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pixman-0.40.0-cjjlunp5xgxyz3ljnl7noclc6zdytess
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libtool-2.4.6-cxbrquzht2yopb7wri2wzkynh5xdl2u7
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxdmcp-1.1.2-eq4r5bsbxmohohk6qxf5jko6mcsgrf3w
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libfontenc-1.1.3-vsqwfpdxuk55r4bsrabohcejfhfom2nf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxau-1.0.8-ct7fnne5xumvy4zmqam6jnqlhgr2w7fq
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libice-1.0.9-f3h5abb2qrcfwwrmcebwo7mxnlnjzfeb
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/freetype-2.10.4-xplpgigqhtfn5c4mo24londcxwduqdi6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gdbm-1.19-67lmhoomilrcwbjpowydn2h2q2vpxaad
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/sqlite-3.34.0-wrallynzub4ezyvmjpon66boyiobnk7m
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libpciaccess-0.16-2zmcrpripc4n5ud6j3t66w5qfi6fn5it
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxcb-1.14-evdxwg5bqrymmb5d23xwbxxjuieazpoi
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libsm-1.2.3-hkvjbdspnuftsvldcqstuykj3rscliyx
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxfont-1.5.2-otb7b6cyfphhmxkrv7vq3ujgcqonnpz6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/mkfontscale-1.1.2-nhxxgc75n7n6o5mfb7hw4dvigjncpa63
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/binutils-2.36.1-r6yhn7ji6u4mqm6almppoqjutfj4fm4k
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-5.32.1-qnrioq3yihe2jlbqkb5m6545r75fje67
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/hwloc-2.4.1-4tofy2py2a3gyg6cimsthzvwuzs6rcog
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/bdftopcf-1.0.5-qtgn4cgjrvlg2ztlfebbfksbnxb27vbx
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/mkfontdir-1.0.7-qo5fnees3igmxwtwg5e5icmiksd43olz
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-test-needs-0.002005-tbgn6rve52ssq6yqhieyxoeblrul4g5m
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/openssl-1.1.1j-bel2tlxdpejk7zzodrwfh4d5745s24au
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-extutils-installpaths-0.012-e4huaqyvsswkcmy7dhweje2jqek3mfys
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-io-html-1.001-j7twhxwqsb4jiv5td2jlauhzsezgtjsl
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-data-dumper-2.173-smtbtxv2zfdgmvc2gvj522egabvsvx2c
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-extutils-config-0.008-hgnzanp75crv4soagorxo2ycbj3bniqc
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libx11-1.7.0-cwf3k6jtbny7ergevoxuntid7fuovxqg
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-try-tiny-0.28-fml2etbflm7hzvahd7fwzysl5jysj36u
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-extutils-helpers-0.026-fp6bwhisqwfekdaqygprrpdgaeqnzifs
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/texinfo-6.5-36mub4xsfvkn4l46stqga5ptjzmclvrk
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-html-tagset-3.20-xcqdldqssfsngzbkgl4a5al2shinbwpd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-encode-locale-1.05-zrw4twfylvdr4mmqpk5tysyttb27ppfh
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-lwp-mediatypes-6.02-bvrjvv46ncysobjoblkcdqnrsxc7c2hz
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-module-build-0.4224-fmcshjxm2kjug5jr7w6lw4mvzvkd4jqj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-http-date-6.02-4rsisneoksgaruqw5ebkknkvihi2ts7d
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/help2man-1.47.16-mmpxwnsjho7fa6x7igixruroftnmakje
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-uri-1.72-mgphufnttjfbe3f6fynsxa2wegxilvjp
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/cmake-3.19.7-tuy42vrxb5plxlrel7tta5n74pytj2pv
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/python-3.8.8-iwpqn7xsqntu2aowb3smgcywnd4wuagl
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxext-1.3.3-jd2uv64wmsmqogw4geq3ekbhw6xcgizd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxt-1.1.5-ypzfmb4jibsnj6wqywm7nj4l4kdt4qa7
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxkbfile-1.0.9-r2wzke2ey2mfnxg6dlbnhyvmngsw2hbw
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxrender-0.9.10-rtkkbyrb7t2gmr2k7l2vbfqqu4p7dnsu
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxfixes-5.0.2-ngt474mcvg5rgxc34kpmoqqogvup4r3l
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gmake-4.3-xcvxb4kabj2hku7mxbvlczpsmqqkrqbd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-html-parser-3.72-im4x42i4v3csjk3dtwqrxtjahy6j5q33
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-module-build-tiny-0.039-3c4a3lvfy5aitikfvbewng4s3oicc6mw
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-file-listing-6.04-25guugqyt4brwl6z2fakqf6ahltl6qjo
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/autoconf-2.69-4o4qgfqme3f53xmvsqw4ai4wlob552wi
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/bison-3.7.4-okbdpbajzflaiffrztaplynjxfozzhji
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-www-robotrules-6.02-ln4pvgsmi5yb2pkjabjgezy7j36noknu
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-http-message-6.13-vjnggjfwkpf6fkamdhc62x5uq2tkcqnj
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-net-http-6.17-4wctz4qbs6vt5waoiy5mua52dshuquej
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libjpeg-turbo-2.0.6-xh33ohuhv76qs5mqh5ronmoremdehl5o
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libssh2-1.8.0-gwerbiduznl6ll53gokxio4t2cd5q2z2
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/py-setuptools-50.3.2-elqr52w4gsa6g2vqsgnvg3m3gtukk5en
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/icu4c-67.1-7xgxsssk6qacmawokelucthykixs3v7r
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/ninja-1.10.2-jipesuyvet7qnss7tvohdkcstprqooxb
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxrandr-1.5.0-xyfuqx676i5wzimdzebqo6aaq4b5y6yr
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxi-1.7.6-xbe2w433kwdtkq3kki2r5cgeizmyxzzr
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/automake-1.16.3-pofjisnhuqwb5pd2batt72u3g7gv3hxp
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/krb5-1.18.2-oafcjthl3mmwag2ruogunmidebssiaqz
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xkbcomp-1.4.4-ofhcxdgugvaybsinzatwxisyexmfon74
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-http-negotiate-6.01-mrlpfmrtenfetp3ed52wynifjco3wsxz
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-http-daemon-6.01-woxbiok5n4cmhdlhflk4nwklwiwdw2lk
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-http-cookies-6.04-4x6xm3o27jkq4d75b7hxg76dwpgmf3xk
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libtiff-4.1.0-colbegfjn6g5muxxpni4xbn4ehnhz64o
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libgit2-1.1.0-zi5us27ai2sbz3qn4fh3ga2bt75g4lpd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/z3-4.8.9-pzk7vbbsf3xazbcacqmzif6rb2qbw2tz
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/py-markupsafe-1.1.1-hft3e3elhzqsjqndlzpmecp7st4t4ini
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/py-pygments-2.6.1-oi4nqjiu5abodc3u44fgb6xjmcksjdr7
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/meson-0.57.1-gir6u4ftbmdkm7xv27sdkmnq6g6eo265
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xrandr-1.5.0-3zzwutwrbcslfdggnlgfzug3schafxv6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxtst-1.2.2-swpp5kg7ea5w5xn53hhm4t2ud3s5dmvo
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/font-util-1.3.2-qe5t3jqamw4f5jdtx76km34dlbrgf7kq
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/findutils-4.6.0-u4upkyiy7ni2aq6dxmppxnpnpvrbm7tq
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/fribidi-1.0.5-pitzkeilgtfxqxkni4vnlqohp54psqpc
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gmp-6.1.2-gmw7p2ssalvzpgc2k6bi26z4tp5apbwa
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/xkbdata-1.0.1-73eysse5dz76jaujevdoh5kbvlz5rkw3
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-libwww-perl-6.33-vj4gbhhw3vmqtlcgghexfvzmrpy65f5r
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/lcms-2.9-gb3t72d4yrqe3kvi3nuucl3jm4f2744b
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/rust-1.48.0-izfbur7fzdjtdljzhkzwzz434epdfe4a
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/llvm-11.0.1-pgyn7wy652bb2newvlrbhjizj534yfug
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/py-mako-1.0.4-f5zwn64viwobpjsm4dtvixmdu7kupijh
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/fontconfig-2.13.1-uielsxlgo3iwzje7gkwbubyrr2smepmo
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/flex-2.6.4-nrogoojtncavetefx4zgijl63itfkwve
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/mpfr-4.0.2-el56i5qd2b6qbzvs44dezolwgpcxn2kd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxkbcommon-0.8.2-az6a6qqddmavapcwesjkhzhnzxj5a63m
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/perl-xml-parser-2.44-hllfv3q447sk45c4eslxemwojvt7v5mw
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/dbus-1.12.8-h5r2py2yly2boxoylrvjrk5nf55xro2f
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxft-2.3.2-geawi7xqvnzvqr7bzqnn2cg66cxtbzkw
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/cairo-1.16.0-umqk6toukclnbcf3a3p4n6zsc3u6vqux
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/mesa-20.2.1-bjn6olo2olcqsdf5c5ersmhsxcbyilvf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gawk-5.1.0-xbx3ihnxwpkqbgqpkuc6s2f4sgjkjhsr
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/intltool-0.51.0-lngfrnyzwwhjui5t53xc7el62q7p2ck3
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gobject-introspection-1.56.1-2fgbneelqabs67jkrbx3w6um6ymhcmqr
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/harfbuzz-2.6.8-spmwwtjieq5ryirwxbk3sknrcphzlo2h
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libepoxy-1.4.3-gwj424wkkp5pfdzega7bf5b3gcavfsuu
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libgpg-error-1.37-36qlxktdkcyaxceqno3cf3xkspuasyr6
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/shared-mime-info-1.9-besm72ftdkwncfvewawdccez7mrrnvxh
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/at-spi2-core-2.38.0-6sjt6ben3johwmmgapruflisz3nsr7r5
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/atk-2.36.0-yvdekspj6npnevs5f25dggyj4f6o2dih
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pango-1.41.0-7keue2fetis2yzfeh7v5ql25k7ewofmf
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libgcrypt-1.9.1-jn5tq7wwce6ccgwvdntqizeihmh6rl5t
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/at-spi2-atk-2.34.2-nh4mga3uzus5s2ixzascvodgaql5pq65
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxslt-1.1.33-wf4djov3n4cqehkjdimgfvkkhjed6r6x
==> Installing gdk-pixbuf-2.42.2-sjt432hndy3hdtjoj42k4oheeuju5b4b
==> No binary for gdk-pixbuf-2.42.2-sjt432hndy3hdtjoj42k4oheeuju5b4b found: installing from source
==> Using cached archive: /opt/ohpc/pub/spack/var/spack/cache/_source-cache/archive/83/83c66a1cfd591d7680c144d2922c5955d38b4db336d7cd3ee109f7bcf9afef15.tar.xz
==> Applied patch /opt/ohpc/pub/spack/var/spack/repos/builtin/packages/gdk-pixbuf/docbook-cdn.patch
==> gdk-pixbuf: Executing phase: 'install'
==> Error: ProcessError: Command exited with status 1:
    'ninja' '-j2' '-v'

1 error found in build log:
     250    [139/148] /opt/ohpc/pub/spack/lib/spack/env/gcc/gcc  -o tests/pixbuf-threads tests/pixbuf-threads.p/pixbuf-threads.c.o tests/pixbuf-threads.p/test-common.c.o -Wl,--as-needed -Wl,--no-undefin
            ed -Wl,-O1 '-Wl,-rpath,$ORIGIN/../gdk-pixbuf' -Wl,-rpath-link,/tmp/root/spack-stage/spack-stage-gdk-pixbuf-2.42.2-sjt432hndy3hdtjoj42k4oheeuju5b4b/spack-src/spack-build/gdk-pixbuf -Wl,--star
            t-group gdk-pixbuf/libgdk_pixbuf-2.0.so.0.4200.2 -lm -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib /opt/ohpc
            /pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/libglib-2.0.so /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/g
            ettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib/libintl.so /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/libgobject-2.0
            .so -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gc
            c-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/libgmodule-2.0.so -pthread -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinq
            oepod6gt6nrqzlc257/lib /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/libgio-2.0.so -Wl,--end-group -Wl,-rpath=/opt/ohpc/
            pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib
     251    [140/148] /opt/ohpc/pub/spack/lib/spack/env/gcc/gcc -Itests/pixbuf-read.p -Itests -I../tests -I. -I.. -Igdk-pixbuf -I../gdk-pixbuf -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/g
            cc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/include/glib-2.0 -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/
            glib-2.0/include -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pcre-8.44-cb3xri3cfopbei3nvowc5pdsvalhy66n/include -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybr
            idge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/include -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libffi-3.3-uinxpvmyxs3kwxjczu5rqdbrowxyzljd/include 
            -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/zlib-1.2.11-u3tjtvurvda5bde33uduo432cg4xgbvj/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winva
            lid-pch -std=c99 -O3 -D_POSIX_C_SOURCE=200809L -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=700 -DHAVE_CONFIG_H=1 -pthread -Wpointer-arith -Wformat=2 -Wstrict-prototypes -Wnested-externs -Wold-style-de
            finition -Wdeclaration-after-statement -Wunused -Wcast-align -Wmissing-noreturn -Wmissing-format-attribute -Wlogical-op -fno-strict-aliasing -Wno-int-conversion -Wno-uninitialized -Wno-disca
            rded-qualifiers -Werror=implicit -Werror=nonnull -Werror=init-self -Werror=main -Werror=missing-braces -Werror=sequence-point -Werror=return-type -Werror=trigraphs -Werror=array-bounds -Werr
            or=write-strings -Werror=address -Werror=int-to-pointer-cast -Werror=pointer-to-int-cast -Werror=empty-body -fvisibility=hidden -MD -MQ tests/pixbuf-read.p/pixbuf-read.c.o -MF tests/pixbuf-r
            ead.p/pixbuf-read.c.o.d -o tests/pixbuf-read.p/pixbuf-read.c.o -c ../tests/pixbuf-read.c
     252    [141/148] /opt/ohpc/pub/spack/lib/spack/env/gcc/gcc  -o tests/pixbuf-read tests/pixbuf-read.p/pixbuf-read.c.o -Wl,--as-needed -Wl,--no-undefined -Wl,-O1 '-Wl,-rpath,$ORIGIN/../gdk-pixbuf' -W
            l,-rpath-link,/tmp/root/spack-stage/spack-stage-gdk-pixbuf-2.42.2-sjt432hndy3hdtjoj42k4oheeuju5b4b/spack-src/spack-build/gdk-pixbuf -Wl,--start-group gdk-pixbuf/libgdk_pixbuf-2.0.so.0.4200.2
             -lm -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/g
            cc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/libglib-2.0.so /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib
            /libintl.so /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/libgobject-2.0.so -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack/lin
            ux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jd
            f3drg/lib/libgmodule-2.0.so -pthread -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib /opt/ohpc/pub/spack/opt/s
            pack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/libgio-2.0.so -Wl,--end-group -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gc
            c-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib
     253    [142/148] /opt/ohpc/pub/spack/lib/spack/env/gcc/gcc -Ithumbnailer/gdk-pixbuf-thumbnailer.p -Ithumbnailer -I../thumbnailer -I. -I.. -Igdk-pixbuf -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-
            sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/include/glib-2.0 -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q
            3jdf3drg/lib/glib-2.0/include -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pcre-8.44-cb3xri3cfopbei3nvowc5pdsvalhy66n/include -I/opt/ohpc/pub/spack/opt/spack/linux-ce
            ntos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/include -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libffi-3.3-uinxpvmyxs3kwxjczu5rqdbrowxy
            zljd/include -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/zlib-1.2.11-u3tjtvurvda5bde33uduo432cg4xgbvj/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64
             -Wall -Winvalid-pch -std=c99 -O3 -D_POSIX_C_SOURCE=200809L -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=700 -DHAVE_CONFIG_H=1 -pthread -Wpointer-arith -Wformat=2 -Wstrict-prototypes -Wnested-externs -
            Wold-style-definition -Wdeclaration-after-statement -Wunused -Wcast-align -Wmissing-noreturn -Wmissing-format-attribute -Wlogical-op -fno-strict-aliasing -Wno-int-conversion -Wno-uninitializ
            ed -Wno-discarded-qualifiers -Werror=implicit -Werror=nonnull -Werror=init-self -Werror=main -Werror=missing-braces -Werror=sequence-point -Werror=return-type -Werror=trigraphs -Werror=array
            -bounds -Werror=write-strings -Werror=address -Werror=int-to-pointer-cast -Werror=pointer-to-int-cast -Werror=empty-body -fvisibility=hidden -DTHUMBNAILER_RETURNS_PIXBUF '-DTHUMBNAILER_USAGE
            ="Thumbnail images"' -MD -MQ thumbnailer/gdk-pixbuf-thumbnailer.p/gnome-thumbnailer-skeleton.c.o -MF thumbnailer/gdk-pixbuf-thumbnailer.p/gnome-thumbnailer-skeleton.c.o.d -o thumbnailer/gdk-
            pixbuf-thumbnailer.p/gnome-thumbnailer-skeleton.c.o -c ../thumbnailer/gnome-thumbnailer-skeleton.c
     254    [143/148] /opt/ohpc/pub/spack/lib/spack/env/gcc/gcc  -o thumbnailer/gdk-pixbuf-thumbnailer thumbnailer/gdk-pixbuf-thumbnailer.p/gdk-pixbuf-thumbnailer.c.o thumbnailer/gdk-pixbuf-thumbnailer.
            p/gnome-thumbnailer-skeleton.c.o -Wl,--as-needed -Wl,--no-undefined -Wl,-O1 '-Wl,-rpath,$ORIGIN/../gdk-pixbuf' -Wl,-rpath-link,/tmp/root/spack-stage/spack-stage-gdk-pixbuf-2.42.2-sjt432hndy3
            hdtjoj42k4oheeuju5b4b/spack-src/spack-build/gdk-pixbuf -Wl,--start-group gdk-pixbuf/libgdk_pixbuf-2.0.so.0.4200.2 -lm -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-1
            0.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/libglib-2.0.so /op
            t/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib/libintl.so /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.
            0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/libgobject-2.0.so -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc2
            57/lib /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/lib/libgmodule-2.0.so -pthread -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack
            /linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5
            q3jdf3drg/lib/libgio-2.0.so -Wl,--end-group -Wl,-rpath=/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gettext-0.21-4npsltkkbkzinqoepod6gt6nrqzlc257/lib
     255    [144/148] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxslt-1.1.33-wf4djov3n4cqehkjdimgfvkkhjed6r6x/bin/xsltproc --nonet --stringparam man.output.quietly 1 --stringp
            aram funcsynopsis.style ansi --stringparam man.th.extra1.suppress 1 --stringparam man.authors.section.enabled 0 --stringparam man.copyright.section.enabled 0 -o docs/gdk-pixbuf-csource.1 htt
            p://cdn.docbook.org/release/xsl/current/manpages/docbook.xsl ../docs/gdk-pixbuf-csource.xml
  >> 256    FAILED: docs/gdk-pixbuf-csource.1
     257    /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libxslt-1.1.33-wf4djov3n4cqehkjdimgfvkkhjed6r6x/bin/xsltproc --nonet --stringparam man.output.quietly 1 --stringparam funcs
            ynopsis.style ansi --stringparam man.th.extra1.suppress 1 --stringparam man.authors.section.enabled 0 --stringparam man.copyright.section.enabled 0 -o docs/gdk-pixbuf-csource.1 http://cdn.do
            cbook.org/release/xsl/current/manpages/docbook.xsl ../docs/gdk-pixbuf-csource.xml
     258    I/O error : Attempt to load network entity http://cdn.docbook.org/release/xsl/current/manpages/docbook.xsl
     259    warning: failed to load external entity "http://cdn.docbook.org/release/xsl/current/manpages/docbook.xsl"
     260    cannot parse http://cdn.docbook.org/release/xsl/current/manpages/docbook.xsl
     261    [145/148] /opt/ohpc/pub/spack/lib/spack/env/gcc/gcc -Ithumbnailer/gdk-pixbuf-print-mime-types.p -Ithumbnailer -I../thumbnailer -I. -I.. -Igdk-pixbuf -I/opt/ohpc/pub/spack/opt/spack/linux-cen
            tos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/include/glib-2.0 -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwe
            tke5q3jdf3drg/lib/glib-2.0/include -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/pcre-8.44-cb3xri3cfopbei3nvowc5pdsvalhy66n/include -I/opt/ohpc/pub/spack/opt/spack/lin
            ux-centos7-sandybridge/gcc-10.2.0/glib-2.66.7-lv7iodnsgekufqdynwetke5q3jdf3drg/include -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libffi-3.3-uinxpvmyxs3kwxjczu5rqdb
            rowxyzljd/include -I/opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/zlib-1.2.11-u3tjtvurvda5bde33uduo432cg4xgbvj/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BI
            TS=64 -Wall -Winvalid-pch -std=c99 -O3 -D_POSIX_C_SOURCE=200809L -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=700 -DHAVE_CONFIG_H=1 -pthread -Wpointer-arith -Wformat=2 -Wstrict-prototypes -Wnested-exte
            rns -Wold-style-definition -Wdeclaration-after-statement -Wunused -Wcast-align -Wmissing-noreturn -Wmissing-format-attribute -Wlogical-op -fno-strict-aliasing -Wno-int-conversion -Wno-uninit
            ialized -Wno-discarded-qualifiers -Werror=implicit -Werror=nonnull -Werror=init-self -Werror=main -Werror=missing-braces -Werror=sequence-point -Werror=return-type -Werror=trigraphs -Werror=
            array-bounds -Werror=write-strings -Werror=address -Werror=int-to-pointer-cast -Werror=pointer-to-int-cast -Werror=empty-body -fvisibility=hidden -MD -MQ thumbnailer/gdk-pixbuf-print-mime-ty
            pes.p/gdk-pixbuf-print-mime-types.c.o -MF thumbnailer/gdk-pixbuf-print-mime-types.p/gdk-pixbuf-print-mime-types.c.o.d -o thumbnailer/gdk-pixbuf-print-mime-types.p/gdk-pixbuf-print-mime-types
            .c.o -c ../thumbnailer/gdk-pixbuf-print-mime-types.c
     262    ninja: build stopped: subcommand failed.

See build log for details:
  /tmp/root/spack-stage/spack-stage-gdk-pixbuf-2.42.2-sjt432hndy3hdtjoj42k4oheeuju5b4b/spack-build-out.txt

==> Warning: Skipping build of gtkplus-3.24.26-ak4zj5c2wgkwo7ljzn4s5pdq36gk4rrf since gdk-pixbuf-2.42.2-sjt432hndy3hdtjoj42k4oheeuju5b4b failed
==> Warning: Skipping build of ghostscript-9.50-elcgd5kia3ngecrhgcx5icv4ykfxvrbi since gtkplus-3.24.26-ak4zj5c2wgkwo7ljzn4s5pdq36gk4rrf failed
==> Warning: Skipping build of groff-1.22.3-tz5z5aka2zmjm7fgzom65xjpr53oo6jk since ghostscript-9.50-elcgd5kia3ngecrhgcx5icv4ykfxvrbi failed
==> Warning: Skipping build of graphviz-2.42.4-ui3pe76s3z4g6rtvmy45gi6ctvc7ittz since groff-1.22.3-tz5z5aka2zmjm7fgzom65xjpr53oo6jk failed
==> Warning: Skipping build of librsvg-2.51.0-kfec3ipvx7fssenhwrrebetay6rxx4zl since gdk-pixbuf-2.42.2-sjt432hndy3hdtjoj42k4oheeuju5b4b failed
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/gtk-doc-1.32-ksxqmzbpx5elludba6xa3rxw7vyzhqvd
[+] /opt/ohpc/pub/spack/opt/spack/linux-centos7-sandybridge/gcc-10.2.0/libcroco-0.6.13-57jxjx37z62kq5isszetexku4fzdy2kz
==> Error: graphviz-2.42.4-ui3pe76s3z4g6rtvmy45gi6ctvc7ittz: Package was not installed
==> Error: Installation request failed.  Refer to reported errors for failing package(s).
```

I can install gdk-pixbuf@2.42.2 on its own. I tried to load this module as well before installing but I get the same error. 
@adamjstewart

--- a/var/spack/repos/builtin/packages/gdk-pixbuf/package.py
+++ b/var/spack/repos/builtin/packages/gdk-pixbuf/package.py
@@ -25,6 +25,8 @@ class GdkPixbuf(Package):
     version('2.31.2', sha256='9e467ed09894c802499fb2399cd9a89ed21c81700ce8f27f970a833efb1e47aa', deprecated=True)
 
     variant('x11', default=False, description="Enable X11 support")
+    # Man page creation was getting docbook errors, see issue #18853
+    variant('man', default=False, description="Enable man page creation")
 
     depends_on('meson@0.55.3:', type='build', when='@2.42.2:')
     depends_on('meson@0.46.0:', type='build', when='@2.37.92:')
@@ -34,8 +36,8 @@ class GdkPixbuf(Package):
     depends_on('shared-mime-info', type='build', when='@2.36.8: platform=cray')
     depends_on('pkgconfig', type='build')
     # Building the man pages requires libxslt and the Docbook stylesheets
-    depends_on('libxslt', type='build')
-    depends_on('docbook-xsl', type='build')
+    depends_on('libxslt', type='build', when='+man')
+    depends_on('docbook-xsl@1.79.2:', type='build', when='+man')
     depends_on('gettext')
     depends_on('glib@2.38.0:')
     depends_on('jpeg')
@@ -47,7 +49,7 @@ class GdkPixbuf(Package):
 
     # Replace the docbook stylesheet URL with the one that our
     # docbook-xsl package uses/recognizes.
-    patch('docbook-cdn.patch')
+    patch('docbook-cdn.patch', when='+man')
 
     def url_for_version(self, version):
         url = "https://ftp.acc.umu.se/pub/gnome/sources/gdk-pixbuf/{0}/gdk-pixbuf-{1}.tar.xz"
@@ -71,6 +73,8 @@ class GdkPixbuf(Package):
         with working_dir('spack-build', create=True):
             meson_args = std_meson_args
             meson_args += ['-Dx11={0}'.format('+x11' in spec)]
+            if '~man' in spec:
+                meson_args += ['-Dman=false']
             meson('..', *meson_args)
             ninja('-v')
             if self.run_tests:

--- a/var/spack/repos/builtin/packages/gdk-pixbuf/package.py
+++ b/var/spack/repos/builtin/packages/gdk-pixbuf/package.py
@@ -72,9 +72,10 @@ class GdkPixbuf(Package):
     def install(self, spec, prefix):
         with working_dir('spack-build', create=True):
             meson_args = std_meson_args
-            meson_args += ['-Dx11={0}'.format('+x11' in spec)]
-            if '~man' in spec:
-                meson_args += ['-Dman=false']
+            meson_args += [
+                '-Dx11={0}'.format('+x11' in spec),
+                '-Dman={0}'.format('+man' in spec),
+            ]
             meson('..', *meson_args)
             ninja('-v')
             if self.run_tests:

